daemon/logger/fluentd/fluentd.go

```diff
@@ -216,6 +221,19 @@ func parseConfig(cfg map[string]string) (fluent.Config, error) {
 		}
 	}
 
+	asyncReconnectInterval := 0
+	if cfg[asyncReconnectIntervalKey] != "" {
+		interval, err := time.ParseDuration(cfg[asyncReconnectIntervalKey])
+		if err != nil {
+			return config, errors.Wrapf(err, "invalid value for %s", asyncReconnectIntervalKey)
+		}
+		if interval != 0 && (interval < minReconnectInterval || interval > maxReconnectInterval) {
+			return config, errors.Errorf("invalid value for %s: value (%q) must be between %s and %s",
+				asyncReconnectIntervalKey, interval, minReconnectInterval, maxReconnectInterval)
+		}
+		asyncReconnectInterval = int(interval)
```

Comment:
this randomly popped into my head @thaJeztah -- I don't think this would function as expected, since the `interval` is a `time.Duration` which will be in [nanoseconds](https://pkg.go.dev/time#Duration). `int(interval.Milliseconds())` would fix this -- https://go.dev/play/p/TJfb0zQ7l3E

Reply:
Ohhh! Great catch. For some reason I was thinking `time.Interval` was in Milliseconds already 🤦 😊😬 

Let me know if you want to update it, or want me to push 👍

Reply:
I figured out the rebasing 🙂 -- updated.

Reply:
At least.. it looks OK 😅

Reply:
Read up a bit more, I remembered the `co-authored-by` field, so I fixed that! Should be good to go again 🙂